index.html

```diff
@@ -126,134 +96,89 @@ <h2 id="background">
-      of this community group. For example, features such as communication
-      across LANs that are out of scope for this community group at this time
-      may be supported through future protocol extensions.
+    </ul>
+    <h3 id="screen-enumeration">
+      Screen Enumeration
+    </h3>
+    <ul>
+      <li>The <a href=
+      "https://github.com/spark008/screen-enumeration/blob/master/EXPLAINER.md">
+        Screen Enumeration API</a> provides developers information about the
+        set of connected physical displays and additional display properties.
+        Use cases include slide show presentation with a laptop screen and a
+        projector, finance applications with multiple windows spread over
+        multiple displays, and windows spanning multiple displays with
+        different hardware properties.
       </li>
-    </ol>
-    <h3 id="scope-apis">
-      New additions to the APIs
+    </ul>
+    <h3 id="window-placement">
+      Window Placement
     </h3>
-    <ol>
-      <li>An addition to the Presentation API called "local presentation mode"
-      is in scope to facilitate 1-UA mode use cases that involve e.g. offline
-      usage, cookie-based authentication, and sharing of application state
-      across the controller and presentation.
+    <ul>
+      <li>The <a href=
+      "https://github.com/spark008/window-placement/blob/master/EXPLAINER.md">
+        Window Placement API</a> builds upon the foundation of the Screen
+        Enumeration API and provides developers further means to open and move
+        windows across the full set of connected displays and control the
+        window state (e.g. maximized, restored). Use cases are shared with the
+        Screen Enumeration.
       </li>
-    </ol>
-    <p>
-      Where possible, the group will use established normative standards as a
-      basis for specifications to reduce the scope of work. For example, local
-      area discovery could be implemented via <a href=
-      "https://tools.ietf.org/html/rfc6762">Multicast DNS</a>.
-    </p>
+    </ul>
     <h2 id="out-of-scope">
       Out of Scope
     </h2>
     <p>
-      For this community group, the following tasks are out of scope.
+      For this Community Group, the following tasks are out of scope:
     </p>
     <ul>
       <li>Any additions to the APIs defined in the Second Screen WG not
-      explicitly listed in <a href="#scope-apis">new additions to the APIs</a>
-      are out of scope other than what is necessary to implement those APIs by
-      remoting requests to the second user agent. Defining media codecs or new
-      specifications for transmitting media codecs is out of scope.
-      </li>
-      <li>Protocols that implement the Remote Playback API for HTML media that
-      uses <a href="https://www.w3.org/TR/encrypted-media/">Encrypted Media
-      Extensions</a> are not in scope.
+      explicitly listed in <a href="#scope-of-work">Scope of Work</a>.
```

Comment:
We discussed some additional features for both APIs at TPAC, but it sounds like you want to make separate changes to the charter to bring those into the CG?

Reply:
I added another bullets noting for both the APIs in scope of WG their v2 features are in scope of the CG linking to GH issues with v2 label for more context.